docs/advanced/document-store/opensearch.md

```diff
@@ -0,0 +1,414 @@
+
+  price=0
+  price=1
+  price=2
+  price=3
+```
+
+### Search by `.text` field
+
+Text search can be easily leveraged in a `DocumentArray` with `storage='opensearch'`.
+To do this text needs to be indexed using the boolean flag `'index_text'` which is set when
+the `DocumentArray` is created  with `config={'index_text': True, ...}`.
+The following example builds a `DocumentArray` with several documents containing text and searches
+for those that have `pizza` in their text description.
+
+```python
+from docarray import DocumentArray, Document
+
+da = DocumentArray(storage='opensearch', config={'n_dim': 2, 'index_text': True})
+with da:
+    da.extend(
+        [
+            Document(text='Person eating'),
+            Document(text='Person eating pizza'),
+            Document(text='Pizza restaurant'),
+        ]
+    )
+
+pizza_docs = da.find('pizza')
+pizza_docs[:, 'text']
+```
+
+will print
+
+```text
+['Pizza restaurant', 'Person eating pizza']
+```
+
+### Search by `.tags` field
+
+Text can also be indexed when it is part of `tags`.
+This is mostly useful in applications where text data can be split into groups and applications might require
+retrieving items based on a text search in an specific tag.
+
+For example:
+
+```python
+from docarray import DocumentArray, Document
+
+da = DocumentArray(
+    storage='opensearch', config={'n_dim': 32, 'tag_indices': ['food_type', 'price']}
+)
+with da:
+    da.extend(
+        [
+            Document(
+                tags={
+                    'food_type': 'Italian and Spanish food',
+                    'price': 'cheap but not that cheap',
+                },
+            ),
+            Document(
+                tags={
+                    'food_type': 'French and Italian food',
+                    'price': 'on the expensive side',
+                },
+            ),
+            Document(
+                tags={
+                    'food_type': 'chinese noddles',
+                    'price': 'quite cheap for what you get!',
+                },
+            ),
+        ]
+    )
+
+results_cheap = da.find('cheap', index='price')
+print('searching "cheap" in <price>:\n\t', results_cheap[:, 'tags__price'])
+
+results_italian = da.find('italian', index='food_type')
+print('searching "italian" in <food_type>:\n\t', results_italian[:, 'tags__food_type'])
+```
+
+will print
+
+```text
+searching "cheap" in <price>:
+  ['cheap but not that cheap', 'quite cheap for what you get!']
+searching "italian" in <food_type>:
+  ['Italian and Spanish food', 'French and Italian food']
+```
+
+````{admonition} Note
+:class: note
+By default, if you don't specify the parameter `index` in the `find` method, the Document attribute `text` will be used
+for search. If you want to use a specific tags field, make sure to specify it with parameter `index`:
+```python
+results = da.find('cheap', index='price')
+```
+````
+
+## Config
+
+The following configs can be set:
```

Comment:
```suggestion
```

This is redundant (I removed the line from other document store docs in a PR earlier today)